sre-recipes/recipes/configs_based/disabled/recipe1.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2021 Google LLC
```

Comment:
FYI: I want to start writing simple tests for the recipes like [this one](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/tree/36902b652d8fad3da96be5591164c74934031e14/tests/recipes). It's probably out of scope for this PR, but we should probably add more before the next release to be safe. Let me know if you have thoughts 

([bug here](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/issues/816))